template.html

```diff
@@ -32,8 +32,44 @@
     />
     <link href="/manifest.json" rel="manifest" crossorigin="use-credentials" />
     <link href="__PUBLIC_PATH__/favicon.ico" rel="shortcut icon" crossorigin="" />
-    <%= htmlWebpackPlugin.tags.headTags.map((tag) => { tag.attributes.crossorigin=''; return tag
-    }).join('') %>
+    <%=
+      htmlWebpackPlugin.tags.headTags.map((tag) => {
+        tag.attributes.crossorigin=''
+        return tag
+      }).join('')
+    %>
+    <!-- Safari sometimes kills the service worker before it can load the initial script -->
+    <script>
+      if ('serviceWorker' in navigator) {
+        function unregisterSW(event) {
+          console.error('Error loading main script', event, event.message)
+          if (event.message === 'Service Worker context closed') {
```

Comment:
I bet this is always false. I don't expect to actually be able to read the event message, hence the log line above. So all we're doing with this PR is logging some load errors.
If we are able to read the message and the `if` is true, then unregistering the current worker and reloading should be the right move. At least a simple manual reload always solves the problem for me.